seedbuilder/areas.ori

```diff
@@ -1862,11 +1916,6 @@ home: MoonGrotto
 		standard-core Dash
 		expert-core WallJump
 		master-lure Bash
-	conn: MoonGrottoBelowTeleporter
```

Comment:
Note that this is *not* the only connection to `MoonGrottoBelowTeleporter`. See line 1835 - there's a master path to reach the "teeth corridor" area using a spider shot from above the teleporter.